src/helpers/AmpConfig.js

```diff
@@ -1,6 +1,9 @@
+const ImageOptimizer = require('./ImageOptimizer');
+
 const AmpConfig = (providedOptions) => {
   const defaultOptions = {
     filter: /.*/,
+    optimizeImages: true,
```

Comment:
are you documenting the idea that we are optimizing images by default anywhere?

Reply:
added a section in the readme